src/main/java/uk/gov/companieshouse/model/dto/dissolution/DissolutionPatchDirectorRequest.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.companieshouse.model.dto.dissolution;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+
+public class DissolutionPatchDirectorRequest {
+
+    @NotBlank
+    @Email
+    private String email;
+
+    @JsonProperty("on_behalf_name")
+    private String onBehalfName;
```

Comment:
is there any validation on the existing on behalf name field in the post? i.e. max length?

Reply:
It is, added to match the one in post